### PR TITLE
Add documentation for missing robots.txt for AB#10088

### PR DESCRIPTION
**Pull request by sslaws**
# Fixes or Implements [AB#10088](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/10088)

* [ ] Enhancement
* [X] Bug
* [ ] Documentation

## Description

During the OpenShift 4 migration the Production robots override was not applied and was not documented.  This PR updates the documentation.

I removed an obsolete folder in the WebClient codebase related to Robots.

I have already run the steps against the Production deployment config.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### Steps to Reproduce

Verify the Production robots.txt file by visiting https:///www.healthgateway.gov.bc.ca/robots.txt default will display a non-prod comment.  

### UI Changes

No

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

File: Apps/WebClient/src/WebClient.csproj [hunk 1]
```diff
@@ -52,7 +52,6 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Views\DumpHeaders\" />
-    <Folder Include="Views\Robots\" />
```
Comment (leobcgov) [inline Apps/WebClient/src/WebClient.csproj:55]: any chances that we have removed this by accident?

File: Tools/BaseBuild/README.md [hunk 2]
```diff
@@ -89,6 +89,25 @@ oc process -f ./adminWebClientSecrets.yaml -p OIDC_SECRET=[Client OIDC Secret]
 oc set env --from=secret/adminwebclient-secrets dc/adminwebclient
 ```
 
+### WebClient Production Only Robots.txt Configuration
+
+Health Gateway has a configurable [robots.txt](../../Apps/WebClient/src/Server/Controllers/RobotsController.cs) that is based on the HealthGateway_Robots.txdt environment variable.  Our default is to have all Robots disallowed in non-Production environments.
```
Comment (leobcgov) [inline Tools/BaseBuild/README.md:94]: is it "..._Robots.txt" or "..._Robot.txdt"

**Review by FuriousLlama — DISMISSED** [review]

**Review by leobcgov — CHANGES_REQUESTED** [review]
few comments/questions

File: Apps/WebClient/src/WebClient.csproj (hunk 1 above)
Comment (sslaws) [reply Apps/WebClient/src/WebClient.csproj:55]: No, I mentioned it in the description of the PR:
_I removed an obsolete folder in the WebClient codebase related to Robots._

File: Tools/BaseBuild/README.md (hunk 2 above)
Comment (sslaws) [reply Tools/BaseBuild/README.md:94]: Probably a nit, but I've resolved.

File: Apps/WebClient/src/WebClient.csproj (hunk 1 above)
Comment (leobcgov) [reply Apps/WebClient/src/WebClient.csproj:55]: nice, my bad for not reading your comments on that :-)

File: Tools/BaseBuild/README.md (hunk 2 above)
Comment (leobcgov) [reply Tools/BaseBuild/README.md]: haha I always have typo in my writing, so I think it helps me to spot these :-)

**Review by leobcgov — APPROVED** [review]

**Review by FuriousLlama — APPROVED** [review]

**Pull request merged**